src/schema/nmdc.yaml

```diff
@@ -796,27 +796,37 @@ slots:
   ecosystem:
     is_a: gold_path_field
     description: >-
-      TODO
+      An ecosystem is a combination of a physical environment (abiotic factors) and all the organisms (biotic factors) that interact with this environment. Ecosystem is in position 1/5 in a GOLD path.
+    comments: The abiotic factors play a profound role on the type and composition of organisms in a given environment. The GOLD Ecosystem at the top of the five-level classification system is aimed at capturing the broader environment from which an organism or environmental sample is collected. The three broad groups under Ecosystem are Environmental, Host-associated, and Engineered. They represent samples collected from a natural environment or from another organism or from engineered environments like bioreactors respectively.
+    see_also: https://gold.jgi.doe.gov/help
 
   ecosystem_category:
     is_a: gold_path_field
     description: >-
-      TODO
+      Ecosystem categories represent divisions within the ecosystem based on specific characteristics of the environment from where an organism or sample is isolated. Ecosystem category is in position 2/5 in a GOLD path.
+    comments: The Environmental ecosystem (for example) is divided into Air, Aquatic and Terrestrial. Ecosystem categories for Host-associated samples can be individual hosts or phyla and for engineered samples it may be manipulated environments like bioreactors, solid waste etc.
```

Comment:
Interestingly, the arity for [comments](https://linkml.io/linkml-model/docs/comments/#slot-comments) is `0..*`. @cmungall Doesn't this mean the `comments` slot should take a list of values?  
It does pass validation though.

Reply:
it should. The schemaloader is liberal in what it accepts, and will convert single value to list for list types. Nevertheless we should conform.